### ISSUE TITLE
A11y_AzureToolsStorageExplorer_AddaresourceviaAzureActiveDirectory_AttachwithAzureAD _ScreenReader: Voiceover does not read the label when focus goes to the Tenant dropdown..

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1977]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver; #Mac; 
Environment Details:
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)
**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Add a resource via Azure Active Directory(Azure AD)" radio button and hit enter on Next button.
4. "Attach with Azure AD" screen gets opened.
5. Start VoiceOver and navigate using Tab key to Tenant dropdown and check whether the Voiceover reads Tenant.
**​Actual:​**
Voiceover does not read the dropdown name as Tenant when Voiceover focus goes to the dropdown.
**Expected:**
VoiceOver should read the dropdown name as Tenant when Voiceover focus goes to the dropdown. 
**UserImpact:**
Screen reader users unable to know the which dialog opened if it will not read the title.
**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference:**
[1977_A11y_AzureToolsStorageExplorer_AddaresourceviaAzureActiveDirectory_AttachwithAzureAD _ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669279/1977_A11y_AzureToolsStorageExplorer_AddaresourceviaAzureActiveDirectory_AttachwithAzureAD._ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo I am unable to repro this bug. See attached gif, where after focus goes to the drop down you can see that VoiceOver says "Tenant:"
![norepro-1977](https://user-images.githubusercontent.com/21112697/68718905-7fa87580-055f-11ea-822e-68b5e425af2d.gif)

**Comment by Suraj1200499** [comment]
@MRayermannMSFT
Verified this issue in Version MacOS: Catalina 10.15
Application Version: 1.11.0 this issue is not reproducing. VoiceOver is announcing the dropdown name as Tenant.

**Comment by MRayermannMSFT** [comment]
Cool. I'm going to close the bug then. Thank you @Suraj1200499 .

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
